addons/mrp_account/models/mrp_production.py

```diff
@@ -12,11 +12,18 @@ class MrpProduction(models.Model):
 
     extra_cost = fields.Float(copy=False, string='Extra Unit Cost')
     show_valuation = fields.Boolean(compute='_compute_show_valuation')
+    wip_production_ids = fields.Many2many('account.move', 'wip_move_production_rel', 'production_id', 'move_id')
+    wip_production_count = fields.Integer("WIP Journal Entry Count", compute='_compute_wip_production_count')
```

Comment:
```suggestion
    wip_move_ids = fields.Many2many('account.move', 'wip_move_production_rel', 'production_id', 'move_id')
    wip_move_count = fields.Integer("WIP Journal Entry Count", compute='_compute_wip_production_count')
```
it points to account moves so it should be move_ids and move_count

Reply:
My mistake, I forgot to properly rename them. Pushed the fixed version now.